### DISCUSSION TITLE
Toggle maximized windows with and without the stage area

### DISCUSSION THREAD

**Discussion by gagarine** [rxhanson/Rectangle#962] [Ideas]
When I use the shortcut to maximise I want my windows to be maximized on the full screen. Yet I also sometimes want to see the stage area. This could be two different shortcut.

But I think it would be best to use the same shortcut and toggle between maximized with and without the stage area visible. The order of the maximized mode could be a preference.

**Comment by rxhanson** [comment]
Makes sense to me. I don't have the bandwidth for this, but I see no reason to turn down a good pull request for the latter scenario you mention, where it's automatically toggled with the same shortcut.

**Comment by jgimeno** [comment]
I was about to write the same, having this option to toggle is totally a nice feature. Is it too difficult to do?

The toggle is the way to go.
